amanuensis/resources/userdatamodel/userdatamodel_message.py

```diff
@@ -25,35 +28,33 @@ def get_all_messages(current_session, logged_user_id):
 
 def get_messages_by_request(current_session, logged_user_id, request_id):
     # messages = current_session.query(Message).filter(Message.sender_id == logged_user_id, Message.request_id == request_id).all()
-    messages = current_session.query(Message).filter(Message.request_id == request_id).all()
+    messages = (
+        current_session.query(Message).filter(Message.request_id == request_id).all()
+    )
     return messages
 
 
-def send_message(current_session, logged_user_id, request_id, subject, body, receivers, emails):
-    '''store message in db and send message to emails via aws ses'''
-    new_message = Message(sender_id=logged_user_id, 
-                        body=body,
-                        request_id=request_id)
+def send_message(
+    current_session, logged_user_id, request_id, subject, body, receivers, emails
+):
+    """store message in db and send message to emails via aws ses"""
+    new_message = Message(sender_id=logged_user_id, body=body, request_id=request_id)
     if receivers:
         new_message.receivers.extend(receivers)
-    
-    
-    if is_env_enabled('SEND_MESSAGE_DEBUG'):
+
+    if is_env_enabled("SEND_MESSAGE_DEBUG"):
         # logger.debug(f"send_message receivers (debug mode): {str(receivers)}")
         logger.debug(f"send_message receivers (debug mode)")
     else:
         if receivers:
             current_session.add(new_message)
             current_session.commit()
 
-    if is_env_enabled('AWS_SES_DEBUG'):
+    if is_env_enabled("AWS_SES_DEBUG"):
         logger.debug(f"send_message emails (debug mode): {str(emails)}")
-    else:
-        if emails:
-            # Send the Messsage via AWS SES
-            send_email_ses(body, emails, subject)
-        
-    return new_message
-
-
+    elif emails:
+        # Send the Message via AWS SES
+        boto_manager_instance = BotoManager(config, logger)
```

Comment:
Botomanager is already initialized at app start (https://github.com/chicagopcdc/amanuensis/blob/b12c3ba50abacbc8f35726ca98aba7ca96def998/amanuensis/__init__.py#L201), you can just get that from the app variables. app.boto.send_email_ses(...